common-ui/src/main/java/com/duckduckgo/mobile/android/themepreview/ui/component/ComponentViewHolder.kt

```diff
@@ -285,6 +286,21 @@ sealed class ComponentViewHolder(val view: View) : RecyclerView.ViewHolder(view)
             val notifyMeView: NotifyMeView = view.findViewById(R.id.notifyMe)
             notifyMeView.setTitle("Component title")
             notifyMeView.setSubtitle("Component subtitle")
+            notifyMeView.setListener(
+                object : NotifyMeListener {
```

Comment:
Dummy implementation as we don't want to grant permissions from the Component Preview.